packages/react/src/components/Alert/Alert.tsx

```diff
@@ -87,22 +87,22 @@ export interface AlertProps
   onDismiss?: ComponentEventHandler<AlertProps>
 
   /**
-   * Called after user's focus.
+   * Called after the alert is focused.
    * @param {SyntheticEvent} event - React's original SyntheticEvent.
    * @param {object} data - All props.
    */
   onFocus?: ComponentEventHandler<AlertProps>
 
-  /** An alert may be formatted to display a successful message. */
+  /** An alert can be formatted to display a successful message. */
```

Comment:
Minor consistency change: "may" and "can" were used interchangeably, so I just picked one.